### PR TITLE
PYTHON-4115 [pymongocrypt] Bundle libmongocrypt 1.8.4

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: bindings/python/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changes in Version 1.8.0
 ------------------------
 
 - Update from manylinux2010 to manylinux2014 wheels.
+- Bundle libmongocrypt 1.8.2 in release wheels.
```
Comment (ShaneHarvey) [inline bindings/python/CHANGELOG.rst:8]: 1.8.2->1.8.4

**Review by ShaneHarvey — COMMENTED** [review]
Should we add rhel-70-arm64 to the test matrix too to make sure it actually works?

File: bindings/python/CHANGELOG.rst (hunk 1 above)
Comment (blink1073) [reply bindings/python/CHANGELOG.rst]: Fixed

**Comment by blink1073** [comment]
> Should we add rhel-70-arm64 to the test matrix too to make sure it actually works?

I added `rhel-82-arm64` and `rhel-70-64-bit` tests.

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**